rct229/rules/section6/section6rule2.py

```diff
@@ -51,18 +51,23 @@ def get_calc_vals(self, context, data=None):
             space_u = context.user
 
             # get allowance
-            lighting_power_allowance_p = table_9_6_1_lookup(
-                space_p["lighting_space_type"]
-            )
+            if space_p["lighting_space_type"] in [
+                GUEST_ROOM,
+                DORMITORY_LIVING_QUARTERS,
+            ]:
+                lighting_power_allowance_p = table_9_6_1_lookup(
+                    space_p["lighting_space_type"]
+                )["lpd"]
+            elif space_p["lighting_space_type"] == DWELLING_UNIT:
```

Comment:
Since the filtering should force the  lighting space type to be DWELLING_UNIT at this point, I would favor logic such as
```
else:
    assert space_p["lighting_space_type"] == DWELLING_UNIT
    lighting_power_allowance_p = 0.6
```
Also, the 0.6 should be a constant defined at the top of the file such as 
```
DWELLING_UNIT_MIN_LIGHTING_POWER_PER_AREA = 06 * ureg("W/ft2")
```